include/kecleon_bros1.h

```diff
@@ -10,17 +10,29 @@ enum KecleonStoreStates
 {
     KECLEON_STORE_INIT = 0,
     KECLEON_STORE_MAIN_MENU,
-    // 2 - 3
+    KECLEON_STORE_INFO,
+    // 3
     KECLEON_STORE_EXIT = 4,
     KECLEON_STORE_NO_STORE_ITEMS,
     KECLEON_STORE_NO_MONEY,
     KECLEON_STORE_TOO_MUCH_MONEY,
-    // 8
-    KECLEON_STORE_NO_ITEMS_TO_SELL = 9,
+    KECLEON_STORE_NO_ITEMS_TO_SELL,
+    KECLEON_STORE_NO_ITEMS,
     KECLEON_STORE_TOO_MANY_ITEMS,
     // 11
     KECLEON_STORE_NOT_ENOUGH_MONEY = 12,
     KECLEON_STORE_CANT_SELL_ITEM,
+    KECLEON_STORE_SELL_ITEM_TOO_MUCH_MONEY,
+    KECLEON_STORE_BUY_ITEM_MENU = 16,
+    KECLEON_STORE_BUY_ITEM_RECEIPT,
+    KECLEON_STORE_BUY_ITEM_INFO = 21,
+    KECLEON_STORE_BUY_ITEM,
```

Comment:
There are enough skips with the enum number values that it may be easier to define all of them explicitly.